Bottom: ad5a844c54538f6ba5fe274edd91fcc2ee399bb0
Top:    cbc46621ad6abc1a41987e35b29406455c6b86d8
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-19 23:28:04 +0000

Refresh of hid-merge-fill-thindraw-_pcb_-

---

diff --git a/src/hid/ps/ps.c b/src/hid/ps/ps.c
index 376be9c..6930507 100644
--- a/src/hid/ps/ps.c
+++ b/src/hid/ps/ps.c
@@ -1496,7 +1496,6 @@ void ps_ps_init (HID *hid)
   hid->do_export          = ps_do_export;
   hid->parse_arguments    = ps_parse_arguments;
   hid->set_layer          = ps_set_layer;
-  hid->fill_pcb_polygon   = ps_fill_pcb_polygon;
   hid->calibrate          = ps_calibrate;
   hid->set_crosshair      = ps_set_crosshair;
 }
@@ -1517,6 +1516,8 @@ void ps_ps_graphics_init (HID_DRAW_API *graphics)
   graphics->fill_circle        = ps_fill_circle;
   graphics->fill_polygon       = ps_fill_polygon;
   graphics->fill_rect          = ps_fill_rect;
+
+  graphics->fill_pcb_polygon   = ps_fill_pcb_polygon;
 }
 
 void
